olive/engine/packaging/packaging_generator.py

```diff
@@ -80,8 +101,13 @@ def _package_candidate_models(
                 temp_resource_path = create_resource_path(model_resource_path.save_to_dir(tempdir, "model", True))
                 # save to model_dir
                 if temp_resource_path.type == ResourceType.LocalFile:
-                    # if model_path is a file, rename it to model_dir / model.onnx
-                    Path(temp_resource_path.get_path()).rename(model_dir / "model.onnx")
+                    if packaging_config.extra_config.get("export_model_in_mlflow_format"):
```

Comment:
Does this say Olive MLFlow output is supported only when model is a single file. What if the model composed model like whisper?

Reply:
It can accept model folder. Changed for both model file and model folder.